waterlog/src/Components/Usage.js

```diff
@@ -33,101 +33,81 @@ class Usage extends Component {
     this.props.fetchCostsDaily();
     this.props.fetchCostsMonthly();
     this.props.fetchCostsSeasonally();
+    this.props.fetchCostsForecastDaily();
+    this.props.fetchCostsForecastMonthly();
   }
 
-  openGraph = graphType => {
+  openGraph = (graphType) => {
     this.setState({
       display: graphType
-    });
-  };
+    })
+  }
 
   getGraphType = () => {
     if (this.state.display === "daily")
-      return (
-        <div>
-          <p className="bold">Cost </p>
-          <DailyCostsReports props={this.props.dailyCost} />
-          <hr id="divide" />
-          <p className="bold">Usage</p>
-          <DailyUsageComponent props={this.props.dailyUsage} />
-        </div>
-      );
+      return <div>
+        <p className="bold">Cost </p>
+        <DailyCostsReports props={this.props}/> 
+        <hr id="divide"/>
+        <p className="bold">Usage</p>
+        <DailyUsageComponent props={this.props.dailyUsage} />
+      </div>
     else if (this.state.display === "monthly")
-      return (
-        <div>
-          <p className="bold">Cost</p>
-          <MonthlyCostsReports props={this.props.monthlyCost} />
-          <hr id="divide" />
-          <p className="bold">Usage</p>
-          <MonthlyUsageComponent props={this.props.monthlyUsage} />
-        </div>
-      );
+      return <div>
+        <p className="bold">Cost</p>
+        <MonthlyCostsReports props={this.props} /> 
+        <hr id="divide"/>
```

Comment:
Ids are supposed to be unique, so if you are reusing it, then make use of classes

Reply:
Changed to class name.